src/applications/facility-locator/containers/VAMap.jsx

```diff
@@ -127,39 +162,40 @@ class VAMap extends Component {
   }
 
   componentDidUpdate(prevProps) {
-    const { currentQuery } = prevProps;
-    const newQuery = this.props.currentQuery;
+    const { currentQuery: prevQuery } = prevProps;
+    const updatedQuery = this.props.currentQuery;
 
     /* eslint-disable prettier/prettier */
-    // A long block of logic AND'ing is easier to read when seen wrapped in a "block"
-    // Yes, it's equivalent without the paren, but understanding is faster with 'em
-    const shouldUpdateSearchQuery = (
+    const shouldZoomOut = ( // ToTriggerNewSearch
```

Comment:
I'm not familiar with how the app works. If this is `true`, does it always result in zooming out, or will it have some other effect given some edge case somewhere?

Reply:
As far as I can tell, yes. It's the only thing that really happens in this lifecycle method.

It either fires `this.zoomOut()` or does the manual adjustment of the map's bounding box to effectively zoom out if on mobile resolutions.

There are a couple locations where some checks are done to see if zooming out should be canceled, however, as noted in the block above canceling appears to have no effect nor is anything currently done to undo the manual "zoom out" for mobile (lines 159 & 195).